Catappult/appcoins-wallet-android
# [APPC-4142]  Scrolling is not clear for all users

**What does this PR do?**

  Removed max size Top Up payment method and increase size in payment methods to make more clear to users scrolling function

**Database changed?**

  No

**How should this be manually tested?**

  Try to make a purchase and topUp in Indonesia to see the behavior of scroll and itens

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4142](https://aptoide.atlassian.net/browse/APPC-4142)

**Code Review Checklist**



[APPC-4142]: https://aptoide.atlassian.net/browse/APPC-4142?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
